openpype/hosts/max/plugins/publish/validate_loaded_plugin.py

```diff
@@ -0,0 +1,125 @@
+        )
+
+        if not required_plugins:
+            return
+
+        invalid = []
+
+        # get all DLL loaded plugins in Max and their plugin index
+        available_plugins = {
+            plugin_name.lower(): index for index, plugin_name in enumerate(
+                get_plugins())
+        }
+
+        for families, plugin in required_plugins.items():
+            families_list = families.split(",")
+            excluded_families = [family for family in families_list
+                                 if instance.data["family"] != family
+                                 and family != "_"]
+            if excluded_families:
+                self.log.debug("The {} instance is not part of {}.".format(
+                    instance.data["family"], excluded_families
+                ))
+                return
+
+            if not plugin:
+                return
+
+            plugin_name = plugin.format(**os.environ).lower()
+            plugin_index = available_plugins.get(plugin_name)
+
+            if plugin_index is None:
+                invalid.append(
+                    f"Plugin {plugin} not exists in 3dsMax Plugin List."
+                )
+                continue
+
+            if not rt.pluginManager.isPluginDllLoaded(plugin_index):
+                invalid.append(f"Plugin {plugin} not loaded.")
+
+        return invalid
+
+    def process(self, instance):
+        invalid_plugins = self.get_invalid(instance)
+        if invalid_plugins:
+            bullet_point_invalid_statement = "\n".join(
+                "- {}".format(invalid) for invalid in invalid_plugins
+            )
+            report = (
+                "Required plugins fails to load.\n\n"
+                f"{bullet_point_invalid_statement}\n\n"
+                "You can use repair action to load the plugin."
+            )
+            raise PublishValidationError(
+                report, title="Required Plugins unloaded")
+
+    @classmethod
+    def repair(cls, instance):
+        # get all DLL loaded plugins in Max and their plugin index
+        available_plugins = {
+            plugin_name.lower(): index for index, plugin_name in enumerate(
+                get_plugins())
+        }
+        required_plugins = (
+            instance.context.data["project_settings"]["max"]["publish"]
+                                 ["ValidateLoadedPlugin"]
+                                 ["family_plugins_mapping"]
+        )
+        for families, plugin in required_plugins.items():
+            families_list = families.split(",")
+            excluded_families = [family for family in families_list
+                                 if instance.data["family"] != family
+                                 and family != "_"]
+            if excluded_families:
+                cls.log.debug("The {} instance is not part of {}.".format(
+                    instance.data["family"], excluded_families
+                ))
+                continue
```

Comment:
I feel like this might be nicer as a message saying something like:

> Instance X does not require plug-in Y, skipping plug-in check

Maybe less informative about what families it checked, but it does explain better what is the resulting behavior.